### PR TITLE
Elm debug support

### LINKED ISSUE #2104: Elm --debug flag for time travelling debugger
# 🙋 feature request

I'd like to be able to see time travelling debugger when using Elm in combination with Parcel. In plain elm it's achieved by passing `--debug` flag to compiler. 

## 💁 Possible Solution

1. It can be enabled by default for all development build unless there is some use case where it's not needed.
2. Or one of Parcel flags can be used to enable this feature

solution 1 - I can do PR for. 
Solution 2 I need some help to figure out which flags can be used.

### PR DESCRIPTION
**Pull request by Maxim-Filimonov**
Fixes #2104

### CODE REVIEW

File: test/elm.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const assert = require('assert');
 const fs = require('../src/utils/fs');
 const {bundle, assertBundleTree, run} = require('./utils');
 
-describe('elm', function() {
+describe.only('elm', function() {
```
Comment (DeMoorJasper) [inline test/elm.js]: Remove this `.only`

File: src/assets/ElmAsset.js [hunk 2]
```diff
@@ -40,10 +40,10 @@ class ElmAsset extends Asset {
       await this.getConfig(['elm.json'], {load: false});
     }
 
+    options.debug = !this.options.minify;
```
Comment (DeMoorJasper) [inline src/assets/ElmAsset.js]: Shouldn't this use `!options.production`

Comment (Maxim-Filimonov) [reply src/assets/ElmAsset.js]: agree 👍

File: test/elm.js (hunk 1 above)
Comment (Maxim-Filimonov) [reply test/elm.js]: Ooops. Good catch

**Comment by Maxim-Filimonov** [comment]
@DeMoorJasper something else is required for merge ?

**Comment by devongovett** [comment]
Due to move to monorepo which rewrote the git history, this PR was automatically closed. Please open a new PR against master with your changes. You can do a diff against the `1.x` branch to see them. Sorry about that!

**Pull request closed without merging**